### PR TITLE
additional badges, fix w3c interop CI and correct the requirements

### PR DESCRIPTION
**Pull request by tsloughter**
I can break this up if needed but figured they were all pretty uncontroversial changes, with the possible exception of the requirements, but I based that on what the CI is running.

### CODE REVIEW

**Review by ferd — APPROVED** [review]

**Comment by ferd** [comment]
Approved but the CI spec seems wrong based on test results:


Run actions/checkout@v2
Error: Invalid repository 'https://github.com/w3c/trace-context'. Expected format {owner}/{repo}.

File: README.md [hunk 1]
```diff
@@ -15,11 +19,11 @@ for a list of features supported.
 
 ## Requirements
 
-- OTP 21.3+
+- Erlang/OTP 22+
 
 If using the Elixir API:
 
-- Elixir 1.8+
+- Elixir 1.11+
```
Comment (bryannaegele) [inline README.md:26]: I think we can say 1.10. Is there something we used that was introduced in 1.11?

https://hexdocs.pm/elixir/1.13/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp

Comment (tsloughter) [reply README.md:26]: I don't know, just basing this on what the lowest version is tested in CI.

Comment (tsloughter) [reply README.md:26]: We can update it later.

**Pull request merged**